pom.xml

```
@@ -32,10 +32,10 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Compiler -->
-        <java.version>1.8</java.version>
-        <java.release>8</java.release>
+        <java.version>11</java.version>
+        <java.release>11</java.release>
         <maven.compiler.source>${java.version}</maven.compiler.source>
-        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.target>${java.release}</maven.compiler.target>
```

```suggestion
        <maven.compiler.release>${java.release}</maven.compiler.release>
```

Since you are no longer supporting compiling with JDK 8 and targeting Java 8, you can remove the `<java.version>`, `<maven.compiler.source>`, and `<maven.compiler.target>` properties and also the `java-release` profile at the end of the pom.

So then you only need to set `<maven.compiler.release>`.